packages/sources/react/src/index.ts

```diff
@@ -23,3 +24,4 @@ export type { VtmnRadioProps } from './components/VtmnRadioButton';
 export type { VtmnTextInputProps } from './components/VtmnTextInput';
 export type { VtmnToggleProps } from './components/VtmnToggle';
 export type { VtmnTooltipProps } from './components/VtmnTooltip';
+export type { VtmnSelectProps } from './components/VtmnSelect';
```

Comment:
Can you also please reorder alphabetically? Thanks!

Reply:
done